drivers/net/ethernet/qualcomm/qca_7k.c

```diff
@@ -1,22 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-2-Clause
```

Comment:
In some changes the multiple line comment style /* */ is used and in other the single line //..

Reply:
This is some strang camp fire knowledge about the SPDX tag. In C files this should be // and in header files it should /* . AFAIK this is because of some kind of tool incompatibility.